#! /bin/sh -e
## Junichi Uekawa <dancer@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use platex instead of latex for compiling.

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1;;
esac

exit 0


--- canna-3.7.orig/doc/man/guide/tex/Imakefile
+++ canna-3.7/doc/man/guide/tex/Imakefile
@@ -3,7 +3,7 @@
 
   CANNAROOT = ../../../..
   INCLUDES  = -I$(CANNAROOT)/include
-  JLATEXCMD = latex
+  JLATEXCMD = platex
   DVI2PSCMD = dvi2ps
   DVIPDFMX  = dvipdfmx
 
